style.css

```diff
@@ -205,28 +234,19 @@ body {
   margin-bottom: 100px;
 }
 
-.logos {
-  margin-bottom: 100px;
-  display: flex;
-  justify-content: center;
-  width: 100%;
-}
-
-.logos > img {
-  padding: 0 20%;
-}
-
 .formsection {
   display: flex;
   flex-direction: column;
   width: 100%;
 }
 
-.formsection input, textarea {
+.formsection input,
+textarea {
   margin-top: 10px;
   line-height: 40px;
   padding-left: 5px;
   font-size: 20px;
+  margin-bottom: 30px;
 }
```

Comment:
- [ ] [OPTIONAL] Please make sure you remove the `line-height` from the `textarea` so that it looks closer to the Figma design.